### DISCUSSION TITLE
Adding "Notes" tab to QSO lightbox

### DISCUSSION THREAD

**Discussion by ofadam** [magicbug/Cloudlog#1333] [Ideas / Feature Requests]
As far as I can tell, you must click on "edit QSO" to see any notes that were added to a logged contact. Would there be any objection to having a tab for any notes on the QSO pop-up lightbox view (ie, click on a callsign in the log)?

Maybe there's a reason this isn't ideal... but I'd love to see what I've written.

**Comment by magicbug** [comment]
No that makes sense might be good under a tab on the popup and only to the user who's logged in.

**Reply by dg9vh** [reply]
I love this idea, too!
